### PR TITLE
add org.localsend.localsend_app

### PR DESCRIPTION
**Pull request by proletarius101**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/localsend/localsend/issues/100
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.localsend.LocalSend.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        for lang in ${FLATPAK_DEST}/localsend/locales/*.pak
+        do
+          locale="$(basename -s .pak $lang)"
+          install -Dm644 -t "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/" "$lang"
+          ln -sf "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/$(basename $lang)" "${FLATPAK_DEST}/localsend/locales/$(basename $lang)"
+        done
+    sources:
+      - type: file
+        dest-filename: LocalSend.AppImage
+        only-arches: [x86_64]
```
Comment (hfiguiere) [inline org.localsend.LocalSend.yml:49]: so you build an aarch64 flatpak with an x86_64 binary.

Comment (hfiguiere) [reply org.localsend.LocalSend.yml:49]: or more like with NO binary.

Comment (JakobDev) [reply org.localsend.LocalSend.yml:49]: The binary is needed. It is a Flutter App, that can't be compiled from source.

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "shared-modules"]
+	path = shared-modules
+	url = https://github.com/flathub/shared-modules.git
```
Comment (JakobDev) [inline .gitmodules:3]: If you comment out libsecret, you no longer need this

File: org.localsend.LocalSend.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+app-id: org.localsend.LocalSend
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
```
Comment (hfiguiere) [inline org.localsend.LocalSend.yml:5]: why ? this is a flutter app.

File: org.localsend.LocalSend.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        sha256: 2434b6e64d484544322d2aedc8dee8b2e7511a3a56c134ea6be794385ca8878f
+      - type: script
+        dest-filename: localsend.sh
+        commands:
+          - set -oue pipefail
+
+          - export FLATPAK_ID="${FLATPAK_ID:-org.localsend.LocalSend}"
+          - export TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
+
+          - exec zypak-wrapper /app/localsend/localsend_app $@
```
Comment (hfiguiere) [inline org.localsend.LocalSend.yml:71]: why this? This is a flutter app.

**Comment by barthalion** [comment]
Is it good to be reviewed? It's still marked as draft.

**Comment by barthalion** [comment]
```
~ 5.9s ❱ flatpak run org.localsend.LocalSend
/app/bin/localsend: line 5: /app/localsend/localsend_app: No such file or directory
```

I assume no.

**Comment by Tienisto** [comment]
I don't know much about Flatpak or AppImage but is this app running in an isolated container (AppImage) inside another isolated container (Flatpak) ?

**Comment by barthalion** [comment]
bot, build org.localsend.localsend_app

**Review by JakobDev — APPROVED** [review]

**Pull request closed without merging**